### DISCUSSION TITLE
Move nextcloud-aio installation to another docker host

### DISCUSSION THREAD

**Discussion by shalak** [nextcloud/all-in-one#5190] [Questions] [answered]
Hello!

I have nextcloud-aio installed on a docker host in Ubuntu VM, and I'd like to migrate it to another docker host that runs in LXC.

What's the best way to migrate everything? The most crucial part, apart from not loosing any data like files, calendar events, etc, is so my family's phones won't re-upload all photos.

**Comment by szaimen** [comment, marked as answer]
Hi, the easiest way would be to create a backup via the AIO interface and then on the new server start a fresh AIO instance and restore the backup in the first step via the AIO interface.

**Reply by shalak** [reply]
Does the AIO backup backs up everything, including AIO composition (selection of talk server, collabora, imaginary, etc) as well as my files for all users? What about cifs-mounted shares (via External Storage) - it won't add those to the backup, right?

**Reply by szaimen** [reply]
Yes, see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#what-is-getting-backed-up-by-aios-backup-solution

**Reply by shalak** [reply]
Hmm, it doesn't mention the AIO composition. If I have a fresh AIO instance without Collabora, will restoring the backup bring back the Collabora?

**Reply by szaimen** [reply]
Ah right. Yes, restoring via the aio interface will also re-enable enabled additional containers
